flutter_local_notifications/lib/src/platform_specifics/darwin/initialization_settings.dart

```diff
@@ -51,6 +52,19 @@ class DarwinInitializationSettings {
   /// Default value is 'false'.
   final bool requestCriticalPermission;
 
+  /// Request permission to provide custom notification settings UI.
+  ///
+  /// When enabled, the system displays a "Configure Notifications in {App}"
```

Comment:
```suggestion
  /// When enabled, the system displays a "Configure Notifications in <application name>"
```

Reply:
Not familiar with suggested changes but lgtm